README.md

```diff
@@ -1,10 +1,11 @@
+:information_source: Note that the `st4000-exploration` branch corresponds to the ST4 version of LSP. If you are looking for information on ST3 version, switch to [master branch](/LSP/tree/master).
```

Comment:
Not sure if that link to master branch will work.

Reply:
https://github.com/sublimelsp/LSP/blob/72aae217a2b4e8dc65e8877f9fe91016ea8b0ca7/README.md 

looks like that it doesn't work